src/main/java/OKRReader.java

```diff
@@ -50,23 +50,19 @@ public static void readOKR() {
         Scanner scanner = new Scanner(System.in);
 
         try (Reader reader = Files.newBufferedReader(Paths.get("C:\\Users\\mercedes.tarrio\\Downloads\\OKR.csv"))) {
-            // columns name
             String[] columns = {"objective", "keyResult1", "keyResult2", "keyResult3", "keyResult4", "keyResult5"};
 
-            // create a mapping strategy
             ColumnPositionMappingStrategy strategy = new ColumnPositionMappingStrategy();
```

Comment:
Refer to the Single Responsibility Principle and reflect on extraction of setups and other non-core actions to private methods.